pipeline/azure-pipelines.yml

```diff
@@ -135,8 +135,9 @@ jobs:
     condition: and(succeeded(), or(eq(variables['ForceWhiteSourceScan'], 'true'), eq(variables['Build.SourceBranch'], 'refs/heads/master')))
     inputs:
       downloadDirectory: $(System.DefaultWorkingDirectory)\Whitesource
-      feedListDownload: 399fb241-ecc7-4802-8697-dcdd01fbb832/69e11f1b-cad7-442a-b676-37722e207221
-      packageListDownload: agent
+      feedsToUse: 'internal'
+      vstsFeed: '399fb241-ecc7-4802-8697-dcdd01fbb832/9e67fb30-4d6b-42ce-93f1-82d875c48b3b'
+      vstsFeedPackage: '1b87817a-74ba-44b2-952f-cb80bf4c6aa6'
```

Comment:
I created most of the yaml by editing the yaml file in the Azure DevOps portal and using the "Yaml Assistant" to create the new yaml. I don't know why the names of the properties are different now, but the property names now match what is in the [MS documentation](https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/package/universal-packages?view=azure-devops).

I added the `feedsToUse: 'internal'` setting to make it clear we're using an internal feed.

I ran a manual build with `ForceWhitesourceScan=true` to check it works (it does: [VS2019](https://dev.azure.com/sonarsource/DotNetTeam%20Project/_build/results?buildId=56387&view=logs&j=ae5f67bc-b508-52d0-18c7-3e0886a35c88&t=aaba8ca1-ef77-5ff9-98ea-409eb392f557) and [VS2022](https://dev.azure.com/sonarsource/DotNetTeam%20Project/_build/results?buildId=56387&view=logs&j=abaf456e-5a9c-50a3-94a8-5c090880e34b&t=b8b0e5b1-6a19-5aae-08cf-ac576f7b97ee))